### PR TITLE
cassandra-driver v3.18.0

### PR DESCRIPTION
**Pull request by anirrudh**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,32 +12,32 @@ source:
...
     - libev  # [not win]
     - msinttypes  # [win and py2k]
 
   run:
     - python
     - numpy
-    - six >=1.6
-    - futures  # [py2k]
+    - six >=1.9
+    - futures<=2.2.0  # [py2k]
```
Comment (scopatz) [inline recipe/meta.yaml]: This needs a single space: `futures <=2.2.0`

Comment (anirrudh) [reply recipe/meta.yaml]: Done :)

**Comment by scopatz** [comment]
ping @petercable - are you OK with this?

**Comment by petercable** [comment]
Yep!

**Comment by petercable** [comment]
@scopatz it's been a long time since I've done anything in conda and a lot has changed. Do I just need to merge this or are there any other steps?

**Comment by scopatz** [comment]
Feel fre to just merge it!

**Pull request merged**